qualtran/bloqs/swap_network.py

```diff
@@ -125,3 +151,12 @@ def build_composite_bloq(
                 )
 
         return {'selection': bb.join(selection), 'targets': targets}
+
```

Comment:
Realizing this is redundant since the decompose should have it.

Reply:
in general: having a known expression as well as a decomposition is more than the sum of its parts because it can provide a consistency check!

Reply:
good point